interfaces/builtin/mir.go

```diff
@@ -47,7 +47,7 @@ const mirPermanentSlotAppArmor = `
 capability sys_tty_config,
 /dev/tty[0-9]* rw,
 
-/{dev,run}/shm/\#* mrw,
+/{dev,run}/shm/\#[0-9]* mrw,
```

Comment:
We should have a comment in here that makes it clear why it is ok. We used to have something for the 15.04 mir-server snap's framework policy, but it was lost along the way.

For future reference, please add:
```
# mir uses open("/dev/shm", O_TMPFILE...) which shows up in AppArmor
# as /dev/shm/#NUMBER. This is then fed to mmap with MAP_SHARED
# for communication with the client (note that the file doesn't actually
# exist and isn't accessible by other processes). Also see:
# https://www.kernel.org/doc/gorman/html/understand/understand015.html
```